### PR TITLE
Split the NovaSeq FC's, fixed the colors, replaced checkboxes with legend

### PR DESCRIPTION
**Pull request by sofiahaglund**
I split the NovaSeq series and did a workaround to fix the colors. Maybe not the cleanest way, but it works :-)
Instead of having the checkboxes for the instrument series, I added a legend so you can just click and de-click the series. I couldn't figure out how to have only the NovaSeq series clicked by default, but maybe it's OK, it's so easy to click and remove the HiSeqs and Miseq?

It's on stage.

### CODE REVIEW

**Comment by ssjunnebo** [comment]
Looks great! I think "color by instrument" might be slightly broken though, but maybe that's not heavily used anyway?

**Comment by alneberg** [comment]
Yeah, I agree with @ssjunnebo, the buttons down to the left doesn't seem to work anymore, and also, HiSeqX an HiSeq should be unchecked by default...

**Comment by sofiahaglund** [comment]
> Yeah, I agree with @ssjunnebo, the buttons down to the left doesn't seem to work anymore, and also, HiSeqX an HiSeq should be unchecked by default...

I did actually notice that they didn't work ^^  but I also thought that it wasn't really used so nobody would notice. Oh well, I can try to fix it...
I did however try to make the Hiseqs unchecked by default but couldn't get it to work, if someone has an idea on how to do it, please let me know :-P

**Comment by sofiahaglund** [comment]
I think I got it to work now :-D 
It's on stage...

File: run_dir/design/flowcell_trend_plot.html [hunk 1]
```diff
@@ -64,32 +64,32 @@ <h1>Flowcells Trend Plot</h1>
...
-                <input id="filter_hiseq" class="filter_inst_type" type="checkbox" name="inst_type_filter" value="D" /> HiSeq
-                <input id="filter_hiseqX" class="filter_inst_type" type="checkbox" name="inst_type_filter" value="E" /> HiSeq X
-                <input id="filter_miseq" class="filter_inst_type" type="checkbox" name="inst_type_filter" value="M" checked /> MiSeq
-                <input id="filter_novaseq" class="filter_inst_type" type="checkbox" name="inst_type_filter" value="A" checked /> NovaSeq
+                <input id="filter_hiseq" class="filter_inst_type" type="checkbox" name="inst_type_filter" value="D"/> HiSeq
+                <input id="filter_hiseqX" class="filter_inst_type" type="checkbox" name="inst_type_filter" value="E"/> HiSeq X
+                <input id="filter_miseq" class="filter_inst_type" type="checkbox" name="inst_type_filter" value="M" /> MiSeq
+                <input id="filter_novaseq" class="filter_inst_type" type="checkbox" name="inst_type_filter" value="A" checked/> NovaSeq
+                <!--<input id="filter_novaseq" class="filter_inst_type" type="checkbox" name="inst_type_filter" value="S4" checked/> -->
```
Comment (sofiahaglund) [inline run_dir/design/flowcell_trend_plot.html]: oooops forgot about this :-D

File: run_dir/static/js/fc_trend_plot.js [hunk 2]
```diff
@@ -156,11 +149,27 @@ function build_series(data, key, name, display_by, filter_inst_type, filter_inst
...
+                }
+            if (data[d].cver.includes('S1')){
+                series_name = "S1";
+                col_color = color_by_s1(series_name)
+                }
+            if (data[d].cver.includes('SP')){
+                series_name = "SP";
+                col_color = color_by_sp(series_name) 
+                }   
+            /*}else{
```
Comment (alneberg) [inline run_dir/static/js/fc_trend_plot.js]: This comment should maybe be removed?

**Comment by alneberg** [comment]
What I can see on stage this morning looks perfect! :)

Some whitespaces are introduced in random places in the code, but I can live with that. There is some commented code that I think should be removed if you don't want it to be used.

Great work!

File: run_dir/static/js/fc_trend_plot.js (hunk 2 above)
Comment (sofiahaglund) [reply run_dir/static/js/fc_trend_plot.js]: yes! :-D

**Pull request merged**